cl/recap/mergers.py

```diff
@@ -644,13 +644,15 @@ def get_or_make_docket_entry(d, docket_entry):
     return de, de_created
 
 
-def add_docket_entries(d, docket_entries, tags=None):
+def add_docket_entries(d, docket_entries, tags=None, create_only=False):
```

Comment:
I think this is ready to merge, but one tweak I'd make is to flip this logic to `do_not_update_existing` instead. I think that'd be more clear, no?

Reply:
Sure, this change is done!